cli/cli_app.go

```diff
@@ -97,6 +97,8 @@ AUTHOR(S):
 
 var MODULE_REGEX = regexp.MustCompile(`module[[:blank:]]+".+"`)
 
+const DEFAULT_TERRAFORM_VERSION_CONSTRAINT = ">= v0.9.3"
```

Comment:
Consider adding comment that this expression will be evaluated by https://github.com/hashicorp/go-version